.circleci/config.yml

```diff
@@ -34,134 +34,138 @@ jobs:
     steps:
       - checkout
       - build_project:
-          path: "angular-instantsearch/loading-indicator"
+          path: 'angular-instantsearch/loading-indicator'
       - build_project:
-          path: "angular-instantsearch/algolia-insights"
+          path: 'angular-instantsearch/algolia-insights'
       - build_project:
-          path: "angular-instantsearch/autocomplete-results-page"
+          path: 'angular-instantsearch/autocomplete-results-page'
       - build_project:
-          path: "angular-instantsearch/debounced-search-box"
+          path: 'angular-instantsearch/debounced-search-box'
       - build_project:
-          path: "angular-instantsearch/extending-widgets"
+          path: 'angular-instantsearch/extending-widgets'
       - build_project:
-          path: "angular-instantsearch/geo-search"
+          path: 'angular-instantsearch/geo-search'
       - build_project:
-          path: "angular-instantsearch/infinite-scroll"
+          path: 'angular-instantsearch/infinite-scroll'
       - build_project:
-          path: "angular-instantsearch/multi-index-autocomplete"
+          path: 'angular-instantsearch/multi-index-autocomplete'
       - build_project:
-          path: "angular-instantsearch/multi-index-hits"
+          path: 'angular-instantsearch/multi-index-hits'
       - build_project:
-          path: "angular-instantsearch/query-suggestions"
+          path: 'angular-instantsearch/query-suggestions'
       - build_project:
-          path: "angular-instantsearch/refresh"
+          path: 'angular-instantsearch/refresh'
       - build_project:
-          path: "angular-instantsearch/routing-basic"
+          path: 'angular-instantsearch/routing-basic'
       - build_project:
-          path: "angular-instantsearch/routing-seo-friendly"
+          path: 'angular-instantsearch/routing-seo-friendly'
       - build_project:
-          path: "angular-instantsearch/routing-state-mapping"
+          path: 'angular-instantsearch/routing-state-mapping'
       - build_project:
-          path: "angular-instantsearch/secured-api-keys"
+          path: 'angular-instantsearch/secured-api-keys'
 
   build_react_projects:
     executor: node14
     steps:
       - checkout
       - build_project:
```

Comment:
something is wrong with the build of react-instantsearch/range-slider-whith-radix-ui (I think it doesn't exist?)

Reply:
Damn typo, fixed in 68fa5357.